README.md

```diff
@@ -160,16 +160,14 @@ The file extension of the template must be one of configured [pageExtensions](ht
 The template, defined in `layouts/docs-page.jsx`, looks like the following:
 
 ```jsx
-export default function Layout(frontMatter) {
-  return ({ children: content }) => {
-    // React hooks, for example `useState` or `useEffect`, go here.
-    return (
-      <div>
-        <h1>{frontMatter.title}</h1>
-        {content}
-      </div>
-    )
-  }
+export default ({ children, frontMatter }) => {
```

Comment:
This must be named otherwise it disables Fast Refresh.

Reply:
Good call, made the change. But I also included this as a comment, let me know if that's too much or whatever.